torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4831,6 +4831,17 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 dtypes=[torch.cdouble]),
                    )),
     OpInfo('add',
+           variant_test_name='without_alpha',
```

Comment:
Why separate add into two OpInfos? Isn't their metadata the same?

Reply:
The idea was to test both without alpha and with alpha versions, because if we test `with_alpha` test variant, then we use the lambda wrapper around `np.add` and `np.multiply`.

This was to just to cover both variants, simple `torch.add` testing vs `np.add` (without alpha), and then with alpha variant.

Though, if it's OK to just test the `with_alpha` variant, we can have only one OpInfo entry for both add and sub in that case. What do you think? @mruberry

Reply:
I think you can test both by defining the reference to include an alpha kwarg with a default value of 1, right?

Reply:
My bad! Thanks for pointing it out, have updated the PR. :)